src/AppBundle/Controller/AddMembreController.php

```diff
@@ -139,123 +177,102 @@ public function HomonymeAction(Request $request)
     public function familleChoiceAction(Request $request) {
 
         $session = $request->getSession();
-
         $nom = $session->get(AddMembreController::MEMBRE_NOM,null);
 
         if(is_null($nom))
         {
+            //restart process
```

Comment:
Il faudrait qu'on prenne l'habitude à chaque fois qu'on fait une action de créer un flash message ($this->addFlash('success', "bravo!"); depuis le controller) histoire que lorsqu'on implémentera un système qui affiche ces alertes, elles soient fonctionnelles et pertinentes directement